scripts/build_locally.py

```diff
@@ -58,12 +58,28 @@ def run(
         cmake_args += [
             "--cmake-executable=" + cmake_executable,
         ]
+
+    process = subprocess.Popen(
```

Comment:
probably it would be good to add a comment to describe the use case when it's still required to pass `-DDpctl_ROOT` option